Assets/Tests/Runtime/ClientServer/ServerObjectManagerTests.cs

```diff
@@ -249,7 +251,10 @@ public void SpawnObjectExceptionTest()
         [Test]
         public void AddPlayerForConnectionFalseTest()
```

Comment:
test names needs to say throws instead of false now

Reply:
fixed